SIS/clarin/views/view-format.xq

```diff
@@ -134,6 +134,21 @@ return
                             {vfm:print-identifiers($format/extId)}
                         </table>
                         
+                        <div> <!-- intentionally slightly indented, to distinguish this fragment without consuming 
```

Comment:
Please be the judge of whether this is the right way to style the content. I wanted to break this monotonous sequence of left-indented lines.